examples/llama.py

```diff
@@ -313,27 +350,81 @@ def greedy_until(self, prompt:str, until, max_length, temperature):
     return output
 
 # **** main code ****
+"""
+test: 
+python3 examples/llama.py  --temperature=0 --count=50 --prompt="Hello."
+output:
+Hello. I'm a 20 year old male. I'm a student at the University of Texas at Austin. I'm a sophomore majoring in Computer Science.
+
+test: 
+python3 examples/llama.py --gen='2' --temperature=0 --count=50 --prompt="Hello."
+output:
+Hello. I'm a 20 year old girl who is looking for a good lay in Palm Coast. I don't care whether it's at your place or not, as long as it's clean.
+
+test:
+python3 examples/llama.py --gen="code" --temperature=0.2 --count=50 --prompt="\
```

Comment:
Shorter tests? I don't understand this \

Reply:
I copied the "\" from the example given in their repo just to make sure we are getting the same outputs as torch. But yes, a short test example would probably be better.